about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/50feea315e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>About - World Chess Tournament</title>
+</head>
+<body>
+    <header>
+        <nav id="navbar">
+            <div id="hamburger">
+                <img id="hamburger-icon" src="images/hamburger-menu-mobile.svg" alt="hamburger-icon">
+                <img id="cancel-icon" src="images/cancel-icon.png" alt="Cancel icon">
+            </div>
+            
+            <a href="/"><img id="fide-logo" src="images/logo-fide.svg" alt="FIDE logo"></a>
```

Comment:
- [ ] Please make sure the logo navigates to the home page. Currently, your logo navigates to a kind of explorer.

**When clicking on the logo**
![image](https://user-images.githubusercontent.com/74411135/235321161-9d5760cb-c312-49ba-9095-9593cdf499c1.png)

This will improve your project accessibly and user experience 😃